Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/debuglog/upload/history/LogUpload.kt

```diff
@@ -1,8 +1,9 @@
 package de.rki.coronawarnapp.bugreporting.debuglog.upload.history
 
+import com.fasterxml.jackson.annotation.JsonProperty
 import java.time.Instant
 
 data class LogUpload(
```

Comment:
I think data models needs to be excluded in proguard rules. See proguard files end

Reply:
LogUpload excluded?
please give some details :-)

Reply:
In release build the annotations is not kept, Please see our progaurd file end. For more information read caveats section here https://github.com/FasterXML/jackson-module-kotlin/blob/2.14/README.md#caveats

Reply:
I still think that it is not neccessary but will add it.
Maybe we should add the following if we observe problems with jackson and proguard:
```
-keepclassmembers public class com.mypackage.** {
    public synthetic <methods>;
}
```